object/MovieDetail.swift

```diff
@@ -0,0 +1,34 @@
+
+import Foundation
+class MovieDetail: Codable {
+    var backgroundImage: String
+    var posterImage: String
+    var title: String
+    var overview: String
+    var releaseDate: String
+    var voteAverage: Float
+    var rateCount: Float
+    var popularity: Float
```

Comment:
Are you sure that all these properties are non-optional? What do you think if you get the API and one of these properties has a nil value?